lib/pages/login/login_page.dart

```diff
@@ -89,13 +109,31 @@ class _LoginPageState extends ConsumerState<LoginPage> {
       return;
     }
     _loading();
-    if (type == Status.login) {
-      await auth.login(_email.text, _password.text, context);
-    } else {
-      await auth.signUp(_email.text, _password.text, context);
-    }
 
-    _loading();
+    try {
+      //
+      if (type == Status.login) {
+        //
+        await auth.login(_email.text, _password.text);
+        //
+      } else {
+        //
+        await auth.signUp(_email.text, _password.text);
+        //
+      }
+
+      _loading(); // stop loader
+
+      if (!mounted) return;
+      await Navigator.pushReplacementNamed(context, HomePage.routename);
```

Comment:
You missed here. This should beneath the line number 117.

And you need to push another screen once you call `signUp` method i.e `CreateProfileScreen` . So this should be below line number 121

Reply:
I was bit thrown off as I was tweaking this. I rechecked this. I thought you were pushing to login account whereas you already created the session. Sorry lemme fix this. Oversight on my part

Reply:
No worries😄